examples/defork/forking_target.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
```

Comment:
The binary `forking_target` should not be in the repo, IMHO. (Can't comment there, so I did it here)

Reply:
I agree, I guess dominik accidentally checked it in. removing it